src/Sulu/Bundle/MediaBundle/Resources/config/storages/local.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
should we really create subfolders here?

I personally prefer suffix `services_storage_local.xml` so I directly see that there are multiple xmls files in the folder registering services.